src/Employer.Web/Orchestrators/DashboardOrchestrator.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Esfa.Recruit.Employer.Web.Services;
+using Esfa.Recruit.Employer.Web.ViewModels;
+using Esfa.Recruit.Storage.Client.Core.Messaging;
+using Esfa.Recruit.Storage.Client.Core.Repositories;
+using System.Threading.Tasks;
+using System.Linq;
+
+namespace Esfa.Recruit.Employer.Web.Orchestrators
+{
+    public class DashboardOrchestrator
+    {
+        private readonly IMessaging _messaging;
+        private readonly IQueryVacancyRepository _queryRepository;
+        private readonly IGetAssociatedEmployerAccountsService _getAccountService;
+
+        public DashboardOrchestrator(IMessaging messaging, IGetAssociatedEmployerAccountsService getAccountsService, IQueryVacancyRepository queryRepository)
+        {
+            _messaging = messaging;
+            _getAccountService = getAccountsService;
+            _queryRepository = queryRepository;
+        }
+
+        public async Task<DashboardViewModel> GetDashboardViewModelAsync(string employerAccountId)
```

Comment:
Obvs this bit will change when QueryStore is introduced.  I'm expecting the returned document for the QS can pretty much be one-to-one mapped to the view model.

Reply:
Yep, I would rather get this work in and then do the QS stuff as a separate piece.